components/NewGoalForm/index.js

```diff
@@ -47,12 +51,13 @@ export default function NewGoalForm({
   handleTargetPerIntervalChange,
   handleAddGoal,
   selectedGoal,
+  handleUserInput,
 }) {
   return (
     <>
       {isModalOpen && (
         <StyledModalBody>
-          <StyledModal hidden>
+          <StyledModal className="formModal" hidden>
```

Comment:
I was wondering why you choose a className here instead of adding the styles to the styled component directly.